rust/ql/test/library-tests/frameworks/postgres/CONSISTENCY/PathResolutionConsistency.expected

```diff
@@ -0,0 +1,17 @@
+multipleMethodCallTargets
+| main.rs:11:5:18:5 | conn.execute(...) | file://:0:0:0:0 | fn execute |
+| main.rs:11:5:18:5 | conn.execute(...) | file://:0:0:0:0 | fn execute |
+| main.rs:22:5:22:37 | conn.execute(...) | file://:0:0:0:0 | fn execute |
+| main.rs:22:5:22:37 | conn.execute(...) | file://:0:0:0:0 | fn execute |
+| main.rs:23:5:23:38 | conn.batch_execute(...) | file://:0:0:0:0 | fn batch_execute |
+| main.rs:23:5:23:38 | conn.batch_execute(...) | file://:0:0:0:0 | fn batch_execute |
+| main.rs:25:5:25:32 | conn.prepare(...) | file://:0:0:0:0 | fn prepare |
+| main.rs:25:5:25:32 | conn.prepare(...) | file://:0:0:0:0 | fn prepare |
+| main.rs:28:5:28:35 | conn.query(...) | file://:0:0:0:0 | fn query |
+| main.rs:28:5:28:35 | conn.query(...) | file://:0:0:0:0 | fn query |
+| main.rs:29:5:29:39 | conn.query_one(...) | file://:0:0:0:0 | fn query_one |
+| main.rs:29:5:29:39 | conn.query_one(...) | file://:0:0:0:0 | fn query_one |
+| main.rs:30:5:30:39 | conn.query_opt(...) | file://:0:0:0:0 | fn query_opt |
+| main.rs:30:5:30:39 | conn.query_opt(...) | file://:0:0:0:0 | fn query_opt |
+| main.rs:35:17:35:67 | conn.query(...) | file://:0:0:0:0 | fn query |
+| main.rs:35:17:35:67 | conn.query(...) | file://:0:0:0:0 | fn query |
```

Comment:
How much of a concern are the new inconsistencies?

Reply:
These consistencies are not really new, they are just surfaced because we can now infer a type for `conn` at https://github.com/github/codeql/blob/bab84d03d14a48d2a1628c38f60d909edefba296/rust/ql/test/library-tests/frameworks/postgres/main.rs#L9.